From 832a47693bfb0a34c45b4012babb894f9a7fff5e Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Mon, 14 Aug 2017 10:48:34 +1000
Subject: [PATCH 2/5] packages: Add pattern to move KDE dev docs to devel

QCH files are used with qt-creator and kdevelop, which are helpful only for
people developing on Solus. If they are building using a lib, they will need
the -devel package installed so they will also get these files. Want to move
them out of "main" as they can be significantly larger than the smaller
libraries themselves. Specific and higher priority to override the
/usr/share/doc rule already included.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 ypkg2/packages.py | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/ypkg2/packages.py b/ypkg2/packages.py
index c9e4b1a..02f23ce 100644
--- a/ypkg2/packages.py
+++ b/ypkg2/packages.py
@@ -210,7 +210,12 @@ class PackageGenerator:
 
         # Vala..
         self.add_pattern("/usr/share/vala*/vapi/*", "devel")
-        self.add_pattern("/usr/share/vala*/vapi/*", "devel")
+
+        # KDE developer documentation
+        self.add_pattern("/usr/share/doc/qt5/*.qch", "devel",
+                         priority=PRIORITY_DEFAULT+1)
+        self.add_pattern("/usr/share/doc/qt5/*.tags", "devel",
+                         priority=PRIORITY_DEFAULT+1)
 
     def add_file(self, path):
         """ Add a file path to the owned list and place it into the correct
-- 
2.14.2

